include/gatekeeper_lls.h

```diff
@@ -217,6 +217,12 @@ struct lls_config {
 };
 
 /*
+ * TODO Find a way to resolve the issue when two or more subscribers will
+ * subscribe the same IP address to the LLS block, and later one of them
+ * requires to clear its request from the LLS.
```

Comment:
Better TODO comment:
```
* TODO Enable an lcore to subscribe multiple times for the same IP address.
* This is needed because it's very hard for the GK blocks avoid duplicates.
* Currently, the LLS block has trouble with unsubscribing those duplicates
* since the LLS block identifies what to remove using
* the pair <IP_address, lcore_id>.
* In addition, the LLS block might not accept a subscription because
* it saves memory for a single subscription for lcore id.
```